packages/components-js/tests/e2e/specs/inline-notification.e2e.ts

```diff
@@ -66,6 +70,17 @@ test('should render without button when dismissButton prop false', async ({ page
   expect(el).toBeNull();
 });
 
+test('should render correct heading tag when heading-tag property is set', async ({ page }) => {
```

Comment:
Wouldn't a generic test based on component-meta information be more helpful to ensure consistency in the future and to reduce maintenance efforts?

Reply:
Added generic test. See `components-with-heading-tag-prop.e2e.ts`